src/main/resources/schema.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE Product (
```

Comment:
I think it's possible to let hibernate create DB table instead of prepare schema by hand. In this approach if you extend entity, you need to remember about updating sql.